### PR TITLE
App info: Create thermostat and modes component

### PR DESCRIPTION
**Pull request by yousefsami**

### CODE REVIEW

File: src/app/layout/app-info/app-info.component.scss [hunk 1]
```diff
@@ -15,4 +15,11 @@
 		width:350px;
 	}
 
+	/deep/.box-title{
```
Comment (torabian) [inline src/app/layout/app-info/app-info.component.scss]: Isn't better to use mixin for this?

File: src/assets/fonts/LICENSE.txt [hunk 2]
```diff
@@ -1,15 +1,6 @@
 Font license info
 
 
-## Web Symbols
```
Comment (torabian) [inline src/assets/fonts/LICENSE.txt]: We are not using web symbols anymore?

Comment (yousefsami) [inline src/assets/fonts/LICENSE.txt]: No worries.

File: src/app/layout/app-info/app-info.component.scss (hunk 1 above)
Comment (yousefsami) [inline src/app/layout/app-info/app-info.component.scss]: ".box-title" just use in the app-info component. no worries

**Pull request merged**